src/redux/books/booksSlice.js

```diff
@@ -1,34 +1,97 @@
-import { createSlice } from '@reduxjs/toolkit';
+/* eslint-disable no-param-reassign */
```

Comment:
- [ ] Please try as much as possible to avoid disabling the eslint ☺️. You need to ensure that you use best Javascript practices and by disabling the eslint, you won't be able to fix the javascript errors.